acceptance/tests/fixtures/cases/sameness/static-server/default/patch.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: static-server
+spec:
+  template:
+    metadata:
+      annotations:
+        "consul.hashicorp.com/connect-inject": "true"
+    spec:
+      containers:
+        - name: static-server
+          image: hashicorp/http-echo:latest
```

Comment:
This should probably be: `docker.mirror.hashicorp.services/hashicorp/http-echo:alpine` or `docker.mirror.hashicorp.services/hashicorp/http-echo:latest` to avoid pull errors in kind.

I think there were problems with `latest` not working on M1 laptops.

Reply:
ah okay, I copied this from existing tests, we might want to go through and change those as well